invenio_collections/percolator.py

```diff
@@ -29,12 +31,16 @@ def new_collection_percolator(target):
     query = IQ(target.dbquery)
     for name in current_search.mappings.keys():
         if target.name and target.dbquery:
-            current_search.client.index(
-                index=name,
-                doc_type='.percolator',
-                id='collection-{}'.format(target.name),
-                body={'query': query.to_dict()}
-            )
+            try:
+                current_search.client.index(
+                    index=name,
+                    doc_type='.percolator',
+                    id='collection-{}'.format(target.name),
+                    body={'query': query.to_dict()}
+                )
+            except TransportError:
```

Comment:
Yes, I don't like the silent error either. If we go for the `try..except..` I will add a `current_app.logger.warn`.

So, the issue I see is that for example, an installation using `invenio-marc-21` will have the following indexes/mappings https://github.com/inveniosoftware/invenio-marc21/tree/master/invenio_marc21/mappings/marc21 and some collection definitions (dbqueries).

Some collection definitions will be appropriate for certain indexes (so no problem in the percolators) and some other ones will not.

So excluding some indexes completely would not solve the problem. Any suggestions? I can otherwise add `logging` in the `try..except` and fix the `kwalitee`